pkgs/games/grapejuice/default.nix

```diff
@@ -15,13 +15,13 @@
 
 python3Packages.buildPythonApplication rec  {
   pname = "grapejuice";
-  version = "3.40.14";
+  version = "3.60.14";
 
   src = fetchFromGitLab {
     owner = "BrinkerVII";
     repo = "grapejuice";
-    rev = "v${version}";
-    sha256 = "1bmkkmi1gx5kc39cjnz5bzwqaicxs0zb6bcv4iny9qccbqf3icrd";
+    rev = "8a86aa31444f6afa97e4ab4cc2c651b1243b8349";
```

Comment:
Why is there no tag for this version?

Reply:
upstream forgets to tag their releases and they dont have issues enabled.